### PR TITLE
feat: [DEL-4999]: Updated UI as per feedback

### PR DESCRIPTION
**Pull request by sonaligoyal-harness**
#### Screenshots

Screenshot 2022-10-04 at 12 34 01 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 1]
```diff
@@ -41,8 +42,9 @@ type delTroubleshoterProps = {
 
 enum InstanceStatus {
   EXPIRED = 'Expired',
-  EXPIRINGIN = 'Expiring In',
-  UPGRADEREQUIRED = 'Upgrade Required'
+  EXPIRINGIN = 'Expiring',
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:45]: Is this a typo?
`EXPIRINGIN = Expiring`
shouldn't it be
`EXPIRING = Expiring`

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 2]
```diff
@@ -41,8 +42,9 @@ type delTroubleshoterProps = {
...
+  LATEST = 'latest',
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:46]: Others are capitalized, but `latest` isn't. Is this intentional?

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx (hunk 1 above)
Comment (sonaligoyal-harness) [reply src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:45]: Will update in follow up PR.

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx (hunk 2 above)
Comment (sonaligoyal-harness) [reply src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:46]: YES, this is intentional and it was provided by UX team only.

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 3]
```diff
@@ -266,36 +268,27 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
...
-      ) === 1
-    : null
   const currentTime = Date.now()
   const isConnected = delegate.activelyConnected
   const text = isConnected ? getString('connected') : getString('delegate.notConnected')
   const status =
     delegate?.delegateGroupExpirationTime !== undefined
-      ? delegate?.delegateGroupExpirationTime < 0 || (!delegate?.immutable && showUpgradeRequired)
-        ? InstanceStatus.UPGRADEREQUIRED
+      ? !delegate?.immutable
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:276]: Reduce code complexity by limiting chained ternary. Better to use a function here with `if-else`

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 4]
```diff
@@ -266,36 +268,27 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
...
-      : delegate?.autoUpgrade
-      ? [Color.GREEN_600, 'AUTO UPGRADE: ON']
-      : [Color.GREY_300, 'AUTO UPGRADE: OFF']
+  const [autoUpgradeColor, autoUpgradeText] = !delegate.activelyConnected
+    ? []
+    : delegate?.autoUpgrade === 'SYNCHRONIZING'
+    ? [Color.ORANGE_400, 'SYNCHRONIZING']
+    : delegate?.autoUpgrade === 'ON'
+    ? [Color.GREEN_600, 'AUTO UPGRADE: ON']
+    : [Color.GREY_300, 'AUTO UPGRADE: OFF']
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:291]: Same here. Ternary chaining should be avoided

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 5]
```diff
@@ -375,30 +369,24 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
 
         {USE_IMMUTABLE_DELEGATE ? (
           <Layout.Horizontal width={columnWidths.instanceStatus} className={css.paddingLeft}>
-            {(!delegate?.immutable && delegate?.groupVersion ? showUpgradeRequired : null) || delegate?.immutable ? (
+            {
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:372]: Remove curly braces as there is no expression

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 6]
```diff
@@ -375,30 +369,24 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
...
+                <Text className={css.statusText} lineClamp={1}>
                   {status}
                 </Text>
-                {status === InstanceStatus.UPGRADEREQUIRED ? (
+                {status === InstanceStatus.LATEST ? (
                   ''
                 ) : delegate.delegateGroupExpirationTime ? (
-                  <ReactTimeago date={delegate.delegateGroupExpirationTime} live />
+                  <div style={{ paddingTop: '2px' }}>
+                    {delegate?.groupVersion?.startsWith('1.0')
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:381]: Ternary not needed. Same can be achieved with
`{!delegate?.groupVersion?.startsWith('1.0') && moment(delegate.delegateGroupExpirationTime).fromNow()}`

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 7]
```diff
@@ -375,30 +369,24 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
...
-                  color={Color.WHITE}
-                  font={{ weight: 'semi-bold', size: 'xsmall' }}
-                  className={css.statusText}
-                  lineClamp={1}
-                >
+                <Text className={css.statusText} lineClamp={1}>
                   {status}
                 </Text>
-                {status === InstanceStatus.UPGRADEREQUIRED ? (
+                {status === InstanceStatus.LATEST ? (
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:377]: Avoid ternary if the condition is is like this.
`{condition ? '' : <Result />`
use `{condition && <Result />}`

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 8]
```diff
@@ -454,22 +442,16 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
               ? getString('connected')
               : getString('delegate.notConnected')
             /*istanbul ignore next */
-            const instanceStatus = instanceDetails?.version?.startsWith('1.0')
-              ? InstanceStatus.UPGRADEREQUIRED
+            const instanceStatus = !delegate?.immutable
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:445]: Avoid ternary chaining.
Also, not sure but I think this `status` logic is repeated. Can this be moved to a util function?

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 9]
```diff
@@ -487,34 +469,20 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
                 </Layout.Horizontal>
                 {USE_IMMUTABLE_DELEGATE ? (
                   <Layout.Horizontal width={columnWidths.instanceStatus} className={css.instanceStatus}>
-                    {((instanceDetails?.version
-                      ? versionText.localeCompare(instanceDetails?.version?.split('.')?.[2]?.substring(0, 3)) === 1
-                      : null) &&
-                      !delegate?.immutable) ||
-                    delegate?.immutable ? (
+                    {
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:472]: Remove curly. No expression here.

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx [hunk 10]
```diff
@@ -487,34 +469,20 @@ export const DelegateListingItem = ({ delegate, setOpenTroubleshoter }: delTroub
...
                           {instanceStatus}
                         </Text>
-                        {instanceDetails.version?.startsWith('1.0') ? (
-                          ''
-                        ) : instanceDetails?.delegateExpirationTime ? (
-                          <ReactTimeago date={instanceDetails.delegateExpirationTime} live />
-                        ) : (
-                          ''
-                        )}
+                        <div style={{ paddingTop: '2px' }}>
```
Comment (depanshu-dhiman) [inline src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx:477]: Same comments as above for this whole block

**Review by depanshu-dhiman — COMMENTED** [review]
Please address

File: src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx (hunk 10 above)
Comment (sonaligoyal-harness) [reply src/modules/30-delegates/pages/delegates/DelegateListingItem.tsx]: @depanshu-dhiman I have resolved some of the comments and for others, I already have ticket for refactoring of this entire component. 
Will address remaining comments in that PR.

**Review by gaurav-jain1 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by depanshu-dhiman — APPROVED** [review]

**Pull request merged**